### PR TITLE
visibility css property

### PR DESCRIPTION
**Pull request by hanbollar**
Fixes https://github.com/Volumetrics-io/mrjs/issues/299

### CODE REVIEW

**Comment by hanbollar** [comment]
still need to do visual check /debug for this pr - one sec

**Comment by hanbollar** [comment]
> from conversation here - ...

another option is to not allow it as css and just have it as a js function (since it's already as part of object3D but this will be easier for discoverability) - but i dont think that's the best approach

**Comment by michaelthatsit** [comment]
Nah it needs to be the CSS property. It's a very commonly used property. 

We could find an alternative to the current solution for hiding mr-app's HTML rendering. maybe setting CSS `opacity: 0`?

**Comment by hanbollar** [comment]
> Nah it needs to be the CSS property. It's a very commonly used property.
> 
> We could find an alternative to the current solution for hiding mr-app's HTML rendering. maybe setting CSS `opacity: 0`?
> 
> This should allow the other elements to use the visibility property

sounds good - swapped to that

this will create the same duplication / inheritance issue that i mentioned above for visibility when people try to play with opacity going forward but that's something we can fix later on - for now this works as a good separation ~

File: src/core/MREntity.js [hunk 1]
```diff
@@ -147,7 +147,21 @@ export class MREntity extends MRElement {
      * @function
      * @description
      */
-    updateStyle() {}
+    updateStyle() {
+        this.setVisibility();
+    }
+
+    setVisibility() {
```
[the remark this answers is not in the document]
Comment (hanbollar) [reply src/core/MREntity.js:154]: sounds good i can swap it into the style system - the next thing i was going to focus on for optimizations was that (after the other two issues highlighted in sync), but i'd rather tackle the separation thing you're mentioning in a separate pr

**Review by michaelthatsit — APPROVED** [review]
The opacity thing feels like a hack but feels like the only solution for now. We can revisit at another point.

**Pull request merged**